src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -252,6 +288,18 @@ def serialize(
                 previous_event.update(
                     {event_key: event.data.get(event_key) for event_key in NODESTORE_KEYS}
                 )
+                if detailed:
```

Comment:
Nit: can this be moved to a new method to keep the core `serialize` method shorter?